src/hotspot/share/logging/logOutputList.cpp

```diff
@@ -68,6 +68,23 @@ LogOutputList::LogOutputNode* LogOutputList::find(const LogOutput* output) const
   return NULL;
 }
 
+void LogOutputList::clear() {
+  // Grab the linked list
```

Comment:
I am not quite sure how the LogOutputList is synchronized. I think you will probably need to call `wait_until_no_readers()` here.